mikupad.html

```diff
@@ -2274,6 +2372,18 @@
 		}
 	}
 
+
+	function adjustWidth(adjustment) {
+
+		var width = document.getElementById("prompt-container").style.maxWidth.replace("em", "").trim();
+		width = String(parseInt(width) + parseInt(adjustment))
+		if (width < 1){
+			width = 1;
+		}
+		document.getElementById("prompt-container").style.maxWidth = width + "em";
+		document.getElementById("modal-width-readout").value = width
+	}

```

Comment:
This code doesn't work when the sidebar is attached to the prompt container.
Besides, it's important to also update the CSS rules within the media queries, such as: `@media (min-width: calc(40.5em + 250px))`. Here, `40.5em` essentially equates to `36em` multiplied by the `fontSize`, which is `1.125rem`.